ClientApp/src/components/admin/dashboard-all/RecipientTable.tsx

```diff
@@ -61,6 +69,25 @@ const DatatableRecipient: React.FC<Props> = ({ data, refreshData, handleRecipien
     return Object.assign({} as RecipientType, recipient);
   };
 
+  const saveComment = () => {
+    if (selectedRecipient) {
+      apiservice
+        .post("admin/recipient/addcomment", {
+          recipientPartitionKey: selectedRecipient.partitionKey,
+          recipientRowKey: selectedRecipient.rowKey,
```

Comment:
```suggestion
          event: selectedRecipient.partitionKey,
          recipientId: selectedRecipient.rowKey,
```

When you rebase on dev after my PR, the selectedRecipient object will have event and recipientId instead of partitionKey and rowKey